nipype/interfaces/traits_extension.py

```diff
@@ -33,7 +33,7 @@
 from traits.api import BaseUnicode
 from traits.api import Unicode
 
-DictStrStr = traits.Dict(str, (bytes, str))
+DictStrStr = traits.Dict((bytes,str,unicode), (bytes,str,unicode))
```

Comment:
Without unicode, it does not work, weirdly my os.environ is all unicode inside nipype, outside it regular python str.
What is weird, I removed my bashrc, then the bug is fixed, I also try to remove a lot of stuff from my bashrc and it is also fixed.
The bashrc is not utf8 coded, and does not contains any special character, so it might be some sourcing that introduce this bug.
So the interaction of these sourced bash scripts with nipype and __future__ is causing os.environ to be all unicode in nipype only, why, I do not know?
Anyway, thanks for the help, I think we can keep the ```DictStrStr = traits.Dict((bytes, str), (bytes,str))``` then ?